liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/AbstractLibertySupport.java

```diff
@@ -604,7 +604,7 @@ private File resolveArtifactFile(org.eclipse.aether.artifact.Artifact aetherArti
                 throw new MojoExecutionException("Unable to resolve artifact: " + aetherArtifact.getGroupId() + ":"
                         + aetherArtifact.getArtifactId() + ":" + aetherArtifact.getVersion());
             }
-        } catch (ArtifactResolutionException e) {
+        } catch (ArtifactResolutionException | NullPointerException e) {
```

Comment:
This will still cause a failure in the reported scenario right? It will just be more graceful? Or does the calling method handle the MojoExecutionException that gets thrown and allows the scenario to succeed?

Reply:
Calling method will catch MojoExecutionException and print a warning that says `"Unable to find the following additional features JSON in the connected repositories: " + mavenCoord + ". Please ignore this warning if this is not a user feature."`